feat(badge): add rectangle shaped badge

tomerla opened the pull request.

lbelinsk commented on src/Badge/Badge.js:
```diff
@@ -53,14 +54,18 @@ Badge.propTypes = {
   ]).isRequired,
 
+  /** set the shape */
+  shape: oneOf(['ellipse', 'rectangle']).isRequired,
```
isRequired is redundant because you set a defaultProp of alipse

tomerla replied:
agreed
but I just follow the existing pattern -  all other props has a `defaultProp` and they are all marked as `isRequired`
do you want me to change them all?

lbelinsk replied:
Yeah, it will be great! I guess it is a trailing error :)

lbelinsk commented on src/Badge/Badge.spec.js:
```diff
@@ -27,4 +27,8 @@ describe('Badge', () => {
     expect(driver.isOfAppearance('H1')).toBeTruthy();
   });
 
+  it('should allow specifying a shape', () => {
```
I would add a test that checks that the default shape is elipse

tomerla replied:
we have already a e2e tests for it
but sure.. why not :)

tomerla commented:
@lbelinsk so I updated a test for a default ellipse shape and updated the docs
please let me know what do you want to do with the `isRequired` (wrote to you in the line comment)
tnx!

tomerla commented:
tnx @lbelinsk !
